tooling/nargo_fmt/src/visitor/expr.rs

```diff
@@ -39,6 +40,16 @@ impl FmtVisitor<'_> {
                     self.format_expr(infix.rhs)
                 )
             }
+            ExpressionKind::Constructor(constructor_expr) => {
+                let type_str = constructor_expr.type_name.to_string();
+            
+                let formatted_fields = constructor_expr.fields
+                    .vecmap(|(field_ident, field_value)| {
+                        format!("{}: {}", field_ident, self.format_expr(field_value.clone()))
+                    })
+                    .join(", ");
```

Comment:
This shouldn't compile (does it?) - `vecmap` is a function and not a method. So you can call it with `vecmap(constructor_expr.fields, |(field_ident, field_value)| ...).join(", ")`